R/uploadPackage.R

```diff
@@ -153,9 +188,42 @@ uploadPackageServer <- function(id) {
       uploaded_pkgs00(uploaded_packages)
     })
     
+    observeEvent(input$rem_pkg_btn, {
+      req(input$rem_pkg_lst)
+      req(user$role == "admin")
+
+      np <- length(input$rem_pkg_lst)
+
+      for (i in 1:np) {
+      pkg_name <- input$rem_pkg_lst[i]
+      dbUpdate(glue::glue("delete from package where name = '{pkg_name}'"), db_name = "database.sqlite")
+      }
+      
+      # run this after all packages in rem_pkg_lst have been removed
+      dbUpdate("delete from package_metrics where package_id not in(select id from package)", db_name = "database.sqlite")
+      dbUpdate("delete from community_usage_metrics where id not in(select name from package)", db_name = "database.sqlite")
+      dbUpdate("delete from comments where id not in(select name from package)", db_name = "database.sqlite")
```

Comment:
Can we make this deletion process a function? I think that would be useful.

Reply:
OK this is now a function called `db_trash_collection`